test/suite/test_cursor_bound03.py

```diff
@@ -148,7 +128,7 @@ def test_bound_general_scenario(self):
         # Test bound api: Test upper bound clearing with only lower bounds.
         self.set_bounds(cursor, 50, "lower")
         cursor.bound("action=clear,bound=upper")
-        self.cursor_traversal_bound(cursor, None, None, self.direction, self.end_key - 50)
+        self.cursor_traversal_bound(cursor, None, None, self.direction, self.end_key - 50 + 1)
```

Comment:
This is a bit odd and should have just be the actual value we subtract which is -49?

Reply:
Yeah thats fair, i guess I wanted to show that self.start_key is a key itself as well. But that is misleading here. I agree lets make it -49.